gulpfile.js/tests/renderPagesFromTemplate.test.js

```diff
@@ -61,7 +61,7 @@ test('renderPagesFromTemplate - renders a section', function (t) {
 
   t.equal(
     $('#sections a:nth-child(2)').attr('href'),
-    '/category-one/index.html',
+    '/category-one' + path.sep + 'index.html',
```

Comment:
Same again here. It's an `href` attribute so should have forward slashes on both platforms